perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -312,6 +312,26 @@ sub inspect : Private {
     $c->stash->{categories} = $c->forward('/admin/categories_for_point');
     $c->stash->{report_meta} = { map { $_->{name} => $_ } @{ $c->stash->{problem}->get_extra_fields() } };
 
+    my @priorities = $c->model('DB::ResponsePriority')->for_bodies($problem->bodies_str_ids)->all;
+    my $priorities_by_category = {};
+    foreach my $pri (@priorities) {
+        my $any = 0;
+        foreach ($pri->contacts->all) {
+            $any = 1;
+            push @{$priorities_by_category->{$_->category}}, $pri->id . '=' . URI::Escape::uri_escape_utf8($pri->name);
+        }
+        if (!$any) {
+            foreach (@{$c->stash->{categories}}) {
+                push @{$priorities_by_category->{$_}}, $pri->id . '=' . URI::Escape::uri_escape_utf8($pri->name);
+            }
+        }
+    }
+    foreach (keys %{$priorities_by_category}) {
+        $priorities_by_category->{$_} = join('&', @{$priorities_by_category->{$_}});
+    }
+
+    $c->stash->{priorities_by_category} = $priorities_by_category;
```

Comment:
There's some deduplication required here, as it's possible for the same priority to appear multiple times (e.g. `<p data-category="Drainage" class="hidden" data-priorities="2=Cat%201A&2=Cat%201A&2=Cat%201A&3=Cat%201B&4=Cat%204">`):
<img width="486" alt="screen shot 2016-12-15 at 15 21 45" src="https://cloud.githubusercontent.com/assets/4776/21229854/dbac1938-c2da-11e6-82ac-d52ed97528b1.png">